config/routes.js

```diff
@@ -32,10 +32,7 @@ module.exports.routes = {
   *                                                                          *
   ***************************************************************************/
 
-  '/*': function log (req, res, next) {
-    sails.log.info(new Date(), req.method, req.url);
-    next();
-  },
+  '/*': { policy: 'angularRouting', skipAssets: true },
```

Comment:
would the `skipRegex` option help here? i.e. `skipRegex: /^\/api\/.*$/` http://stackoverflow.com/questions/22770035/how-to-setup-sails-js-routes-to-support-pushstate-with-a-spa-on-the-frontend

Or would you still need to check in the policy anyway?